client/history/peerResolver.go

```diff
@@ -29,15 +29,25 @@ import (
 // PeerResolver is used to resolve history peers.
 // Those are deployed instances of Cadence history services that participate in the cluster ring.
 // The resulting peer is simply an address of form ip:port where RPC calls can be routed to.
-type PeerResolver struct {
+//
+//go:generate mockgen -package $GOPACKAGE -source $GOFILE -destination peerResolver_mock.go -package history github.com/uber/cadence/client/history PeerResolver
```

Comment:
please rename this file to peer_resolver.go and the mock file to peer_resolver_test.go. we keep fixing naming inconsistencies as we touch